File: Project/Backend/src/ArtAuction.Application/DTOs/ArtworkPost/ArtistArtworkPostDto.cs
using Microsoft.AspNetCore.Http;

namespace ArtAuction.Application.DTOs.ArtworkPost;

public class ArtistArtworkPostDto
{
    // Attributes
    public int Id { get; set; }
    public string Title { get; set; }
    public string Description { get; set; } 
    public decimal InitialPrice { get; set; }
    public decimal BuyNewPrice { get; set; }
    public DateTime StartDate { get; set; }
    public DateTime EndDate { get; set; }
    public string Image { get; set; }
    public string CategoryName { get; set; }
    public int adminId { get; set; }
    public string[] Tags { get; set; }
}

File: ArtworkPostServices/src/main/java/com/artauction/artworkpostservices/Dtos/ArtworkPost/ArtistArtworkPostDto.java
package com.artauction.artworkpostservices.Dtos.ArtworkPost;

import lombok.Getter;
import lombok.Setter;

import java.math.BigDecimal;
import java.time.LocalDateTime;

@Setter
@Getter
public class ArtistArtworkPostDto {
    private Integer id;
    private String title;
    private String description;
    private BigDecimal initialPrice;
    private BigDecimal buyNowPrice;
    private LocalDateTime startDate;
    private LocalDateTime endDate;
    private String image;
    private String categoryName;
    private Integer adminId;
    private String[] tags;
}
